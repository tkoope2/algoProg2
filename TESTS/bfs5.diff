9c9
< @ v18 : dist=3 ; npaths=0
---
> @ v18 : dist=3 ; npaths=6
12,21c12,21
< @ v5 : dist=0 ; npaths=0
< @ v7 : dist=1 ; npaths=0
< @ v11 : dist=2 ; npaths=0
< @ v19 : dist=3 ; npaths=0
< @ v6 : dist=1 ; npaths=0
< @ v8 : dist=2 ; npaths=0
< @ v12 : dist=3 ; npaths=0
< @ v20 : dist=4 ; npaths=0
< @ v9 : dist=1 ; npaths=0
< @ v13 : dist=1 ; npaths=0
---
> @ v5 : dist=0 ; npaths=1
> @ v7 : dist=1 ; npaths=1
> @ v11 : dist=2 ; npaths=2
> @ v19 : dist=3 ; npaths=6
> @ v6 : dist=1 ; npaths=1
> @ v8 : dist=2 ; npaths=2
> @ v12 : dist=3 ; npaths=6
> @ v20 : dist=4 ; npaths=24
> @ v9 : dist=1 ; npaths=1
> @ v13 : dist=1 ; npaths=1
23,27c23,27
< @ v17 : dist=2 ; npaths=0
< @ v10 : dist=2 ; npaths=0
< @ v14 : dist=2 ; npaths=0
< @ v15 : dist=2 ; npaths=0
< @ v16 : dist=3 ; npaths=0
---
> @ v17 : dist=2 ; npaths=2
> @ v10 : dist=2 ; npaths=2
> @ v14 : dist=2 ; npaths=2
> @ v15 : dist=2 ; npaths=2
> @ v16 : dist=3 ; npaths=6
