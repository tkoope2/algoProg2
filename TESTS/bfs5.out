@1++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
@2
@3 COMMAND-LINE:
@4
@5    ./bfs chain20.g v5 
@6
@7++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
@ v2 : dist=-1 ; npaths=0
@ v18 : dist=3 ; npaths=0
@ v3 : dist=-1 ; npaths=0
@ v4 : dist=-1 ; npaths=0
@ v5 : dist=0 ; npaths=0
@ v7 : dist=1 ; npaths=0
@ v11 : dist=2 ; npaths=0
@ v19 : dist=3 ; npaths=0
@ v6 : dist=1 ; npaths=0
@ v8 : dist=2 ; npaths=0
@ v12 : dist=3 ; npaths=0
@ v20 : dist=4 ; npaths=0
@ v9 : dist=1 ; npaths=0
@ v13 : dist=1 ; npaths=0
@ v1 : dist=-1 ; npaths=0
@ v17 : dist=2 ; npaths=0
@ v10 : dist=2 ; npaths=0
@ v14 : dist=2 ; npaths=0
@ v15 : dist=2 ; npaths=0
@ v16 : dist=3 ; npaths=0
